pepper-apis/src/main/java/org/broadinstitute/ddp/util/I18nUtil.java

```diff
@@ -133,9 +133,9 @@ public static Locale resolvePreferredLanguage(@Nullable Locale preferredLocale,
         if (null != preferredLocale) {
             LanguageRange preferred = new LanguageRange(preferredLocale.toLanguageTag(), LanguageRange.MAX_WEIGHT);
```

Comment:
Should this be `MIN_WEIGHT` instead to lower its priority?

Reply:
Changing it this way apparently makes the Locale.lookup to prefer the fallback language over the preferred one from the user profile - breaking tests down. I would leave it as-is unless you are adamant to change it.

Reply:
Interesting. It should have the same weight as the fallback, but comes before the fallback in the priority list. Let's leave it for now then.